pallets/staking/src/lib.rs

```diff
@@ -949,6 +946,10 @@ pub trait Config:
     /// their reward. This used to limit the i/o cost for the nominator payout.
     type MaxNominatorRewardedPerValidator: Get<u32>;
 
+		/// The fraction of the validator set that is safe to be offending.
+		/// After the threshold is reached a new era will be forced.
+		type OffendingValidatorsThreshold: Get<Perbill>;
```

Comment:
fix indent
```suggestion
    /// The fraction of the validator set that is safe to be offending.
    /// After the threshold is reached a new era will be forced.
    type OffendingValidatorsThreshold: Get<Perbill>;
```

Reply:
We have auto-formatting turned off for the Staking pallet, so that it is easier to compare with Substrate's code.  This is why formatting here doesn't fail the `lint` test in our CI pipeline.

Reply:
hmm.  Fixing indent should be fine, but it has to be done manually, otherwise some single line bits of code get re-formatted to multi-line code (or vice versa).